### PR TITLE
Implemented text blinking

### PR DESCRIPTION
**Pull request by basdxz**
Implemented blinking when using ANSI escape characters for both slow and rapid blinking.

Disabled by default, with preset values of 1000ms for slow and 500ms for rapid.

Blinking is done by inverting the text style and triggering a repaint.

### CODE REVIEW

File: ui/src/com/jediterm/terminal/ui/TerminalPanel.java [hunk 1]
```diff
@@ -1221,13 +1239,82 @@ void setShape(CursorShape shape) {
...
+      long rapidRate = getRapidTextBlinkingPeriod();
+      long rapidTriggerPoint = currentTime - rapidRate;
+      if (myLastRapidTextBlink < rapidTriggerPoint) {
+        myRapidTextBlinking = !myRapidTextBlinking;
+        if (myLastRapidTextBlink == 0)
+          myLastRapidTextBlink = currentTime;
+        else while (myLastRapidTextBlink < rapidTriggerPoint)
+          myLastRapidTextBlink += rapidRate;
+        repaint = true;
+      }
```
Comment (segrey) [inline ui/src/com/jediterm/terminal/ui/TerminalPanel.java:1283]: Could you please try to get rid of the code duplication? Maybe have a dedicated class managing blinking of a particular period?

Comment (basdxz) [reply ui/src/com/jediterm/terminal/ui/TerminalPanel.java]: Dedicated class added now, although I'm not sure if de-duplicating code here makes much sense considering that there would only ever be two type of blinking.

**Comment by Desoroxxx** [comment]
Any news on this?

**Comment by segrey** [comment]
Merged as 78b143010fc53456f2d16eb67572ed23b4a99543. Thank you!

**Pull request closed without merging**